restifizer/RestifizerResponse.h

```diff
@@ -0,0 +1,22 @@
+//
+//  RestifizerResponce.h
+//  Restifizer
+//
+//  Created by Nikita Pankiv on 4/29/16.
+//  Copyright © 2016 VedideV. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RestifizerError.h"
+
+@class RestifizerRequest;
+
+@interface RestifizerResponse : NSObject
+
+@property (nonatomic, strong) NSData *contentData;
```

Comment:
Because of this method
`(void)connection:(NSURLConnection *)connection didReceiveData:(NSData *)data`
and there is no reason in modifying private-layer, in the end - the results will be the same.